### DISCUSSION TITLE
Taxes aren't working

### DISCUSSION THREAD

**Discussion by haf** [swellstores/community#100] [Help] [answered]
Taxes in Europe with EU-internal / EU-external don't work. VAT is added in checkout instead of to the price displayed in the shop, there's no "fill in your VAT number to do reverse VAT" and it's called "Sales tax" and not VAT. I've tried all combinations of settings in the pricing page.

**Comment by ericingram** [comment, marked as answer]
There's no mechanism to show VAT in the product page, only in the cart. You could use translation text to change the name from "Sales tax" to "VAT", but I agree it should use the name of the tax rule instead of defaulting to the translation text.

Also true that our hosted checkout does not currently support entry of a VAT number, however the API does support it if you wanted to build a custom checkout.
